### PR TITLE
Riagrawal-Sanjose added a new .md file

### LINKED ISSUE #17: Create a new file for riagrawal San Jose, CA
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Collaborate
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by riagrawal**
closes #17

### CODE REVIEW

**Comment by omkargaikwad** [comment]
🐙

File: riagrawal-sanjose.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# San Jose, CA
+
+#### Places to eat
+- Erawan Thai
```
Comment (omkargaikwad) [inline riagrawal-sanjose.md:4]: where is it ?

**Review by chrissanjivxavier — APPROVED** [review]
Thank you for the places mentioned, will check it out

**Review by omkargaikwad — APPROVED** [review]
I approve

**Pull request merged**